jobs/rocoto/atmos_products.sh

```diff
@@ -15,21 +15,13 @@ if (( status != 0 )); then exit "${status}"; fi
 export job="atmos_products"
 export jobid="${job}.$$"
 
-###############################################################
-# shellcheck disable=SC2153,SC2001
-IFS='_' read -ra fhrs <<< "${FHRLST//f}" # strip off the 'f's and convert to array
+# Negatation needs to be before the base
+fhr3_base="10#${FHR3}"
+export FORECAST_HOUR=$(( ${fhr3_base/10#-/-10#} ))
```

Comment:
The `#` in this regex breaks vim syntax highlighting unless it is escaped.
```suggestion
export FORECAST_HOUR=$(( ${fhr3_base/10\#-/-10\#} ))
```

Reply:
Not sure about making the code harder to read everywhere else to accommodate faulty parsers, even ones as widely used as `vim`.